content/en/docs/Responses/AlbumID3.md

```diff
@@ -59,12 +105,38 @@ description: >
 | `genre` | `string` | No |     | The album genre|
 | `played` | `string` | No | **Yes**    | Date the album was last played. [ISO 8601]|
 | `userRating` | `int` | No | **Yes**    | The user rating of the album. [1-5]|
+| `recordLabels` | Array of `string` | No |  **Yes**   | The labels producing the album. |
```

Comment:
I guess recordLabels should follow the same pattern as genres? This will allow future list/filter api for record labels like genres.

Reply:
Well at this point we can too, but will any server ever use that and expose ids? I'm not even sure any server will store more than 1 value.